client/src/components/DropdownMenu/Dropdown.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './DropdownStyles.css';
```

Comment:
let's use styled-components for these styles, should be easy to change without having to think too much about it. I see that you are using it in the Navbar component quite successfully.
Importing styles like this has the downside that they aren't scoped and thus global. Nice to be consistent with how we do things anyway.

Reply:
I agree but I'm having a hard time converting the css I wrote to styled components. I just tried again and couldn't do it right. I'm going to leave it as is for now.